sdk/communication/Azure.Communication.JobRouter/src/Models/UpdateJobOptions.cs

```diff
@@ -50,7 +50,7 @@ public UpdateJobOptions(string jobId)
         public string ChannelId { get; set; }
 
         /// <summary> The priority of this job. </summary>
-        public int Priority { get; set; }
+        public int? Priority { get; set; }
```

Comment:
It was recommended to not have nullable types in public models. Should we set a default value of -1 instead?

Reply:
This would be applicable for all `int?`

Reply:
As discussed offline, according to SDK review board, nullable value types are ok since, there's no backwards compatibility issue